runners/flink/pom.xml

```diff
@@ -82,7 +82,7 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-surefire-plugin</artifactId>
           <configuration>
-            <argLine>-Dlog4j.configuration=log4j-test.properties  -XX:-UseGCOverheadLimit</argLine>
+            <argLine>-Dlog4j.configuration=log4j-test.properties  -XX:-UseGCOverheadLimit -Xmx512m</argLine>
```

Comment:
The hope:

* Define a `-Dbeam.argline=....` ENV in `.travis.yml`
* Dereference `${beam.argline}` in the three places it needs to be right now.
* Separate out the possibility of cleaning up flink override of argline.